scripts/get-version.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+
+"""
+Returns the version used in this project of the tool provided as a CLI argument.
+"""
+
+import sys
+
+if len(sys.argv) != 2:
+    sys.exit(sys.argv[0] + " <TOOL>")
+
+tool = sys.argv[1].strip()
+
+TOOLS_USED = {
```

Comment:
This could also be a JSON file, then we don't need have a script but can use `jq`.

Also, perhaps if we had it in the some JSON file we could use that for the constants as well (where we specify the cairo version, but will also specify a devnet version in the future)

Reply:
At first it was a JSON parsed with `jq`, but then I realized that way we would need to have the `jq` logic in multiple places.

Reply:
As agreed, this can stay as it is and will be handled in a different PR.